backend/src/Service/SendAttempt/SendAttemptService.php

```diff
@@ -56,6 +58,12 @@ public function handleAfterSendAttempt(SendAttempt $sendAttempt): void
                     SuppressionReason::BOUNCE,
                     $parser->getFullMessage()
                 );
+                $this->infrastructureBounceService->createInfrastructureBounce(
```

Comment:
wrong, this is creating an infrastructure bounce when `isRecipientBounce()`. Check `isInfrastructureError` separately and create a record